configure

```diff
@@ -718,26 +715,12 @@ def configure_v8(o):
 def configure_openssl(o):
   o['variables']['node_use_openssl'] = b(not options.without_ssl)
   o['variables']['node_shared_openssl'] = b(options.shared_openssl)
-  o['variables']['openssl_no_asm'] = (
-    1 if options.openssl_no_asm else 0)
+  o['variables']['openssl_no_asm'] = 1 if options.openssl_no_asm else 0
 
   if options.without_ssl:
     return
 
-  if options.shared_openssl:
-    (libs, cflags) = pkg_config('openssl') or ('-lssl -lcrypto', '')
-
-    libnames = options.shared_openssl_libname.split(',')
-    o['libraries'] += ['-l%s' % s for s in libnames]
-
-    if options.shared_openssl_libpath:
-      o['libraries'] += ['-L%s' % options.shared_openssl_libpath]
-
-    if options.shared_openssl_includes:
-      o['include_dirs'] += [options.shared_openssl_includes]
-    else:
-      o['cflags'] += cflags.split()
-
+  configure_library('openssl', o)
```

Comment:
I personally don't care that much about pep8.  As long as changes are not too incongruent with existing code, they're fine by me.